specs/lib/triBoxOverlapSpec.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+var Cesium = require('cesium');
+var Cartesian3 = Cesium.Cartesian3;
+var triBoxOverlap = require('../../lib/triBoxOverlap');
+
+describe('triBoxOverlap', function() {
```

Comment:
These tests are new, and they don't cover every possible return path although they cover all the helper functions. Do you have comprehensive tests? Otherwise I could write them, but figuring out the geometry for each case could take some time.